deployment/.gitignore

```diff
@@ -0,0 +1,11 @@
+*
+!.gitignore
+!Makefile
+!active
+!*/
+!/dev/**
+!/prod/**
+**/gcp-credentials.json
+**/application_secrets.json
+**/.terraform
+**/local_variables.tf
```

Comment:
I would caution against structuring the .gitignore file this way - I think it will lead to confusion. Are you modeling this after some Terraform best practice?